.github/workflows/ci.yaml

```diff
@@ -152,8 +151,87 @@ jobs:
             -c ../rust --dart-input lib/main.web.dart --root web/ --run-tests
           current_path: frb_example/pure_dart/dart
 
+  dart_web_bench:
```

Comment:
just to assert that they work fine, avoid regression.
also they are run without `--release` in `ci`, but with it on `gh-pages`: initially it was to save some time on builds, but given it's cached maybe it can be `--release` everywhere?
what is your opinion ?

Reply:
Then maybe all on ci.yaml?

Reply:
In addition, would be great to run benchmark on *all* PR as well (like the spirit of https://github.com/marketplace/actions/criterion-compare-prs)- just do not *submit* those to main branch.

Reply:
> In addition, would be great to run benchmark on all PR as well (like the spirit of https://github.com/marketplace/actions/criterion-compare-prs)- just do not submit those to main branch.

feel free to do it in a subsequent PR :)

Reply:
Well, now that I realize what you really want, if it's just a couple of refactoring + bench loop I'm gonna look into it.
I just want to limit the time allocated here since there's this another PR waiting (refactoring github workflows and just commands) and I'd like to get back at my app asap too.